pkg/apis/submariner.io/v1/types.go

```diff
@@ -344,3 +344,51 @@ type GlobalIngressIPList struct {
 
 	Items []GlobalIngressIP `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:shortName="gwrt"
+
+type GWRoute struct {
```

Comment:
Let's spell it out for clarity.

```suggestion
type GatewayRoute struct {
```